ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureBlobDataAccessor.java

```diff
@@ -120,6 +121,11 @@
     this.purgeBatchSize = AzureCloudConfig.DEFAULT_PURGE_BATCH_SIZE;
   }
 
+  /** Visible for testing */
+  void setUpdateCallback(Callable callback) {
```

Comment:
use generics in both the member variable and method arg. If the return type of call doesn't matter, you can use `Callable<?>` in both places.